v-next/hardhat-build-system/package.json

```diff
@@ -52,6 +52,7 @@
     "p-map": "^4.0.0",
     "resolve": "1.17.0",
     "semver": "^7.6.2",
+    "solc": "0.7.3",
```

Comment:
How should I think about the solc version we would use here. We are porting the `hardhat-core` version here, but shouldn't we be pulling in the latest version (and keeping it up to date)?

Reply:
I just deferred the decision tbh. IIRC in v2 we pull the one that's part of the default config, but that doesn't add a ton of value, as you are gonna want to download the native compiler anyway.